inc/media/namespace.php

```diff
@@ -79,7 +79,9 @@ function add_lord_of_the_files_types( $existing_mimes = [] ) {
 	$lord_of_the_files_activated = ( is_plugin_active_for_network( 'blob-mimes/index.php' ) || is_plugin_active( 'blob-mimes/h5p.php' ) ) && class_exists( 'blobfolio\\wp\\bm\\mime\\aliases' );
 	if ( $lord_of_the_files_activated ) {
 		foreach ( unknown_upload_types( $existing_mimes ) as $k => $v ) {
-			if ( isset( \blobfolio\wp\bm\mime\aliases::TYPES[ $k ] ) ) {
+			if ( $k === 'nlogo' )
```

Comment:
Check to see if the file extension if nlogo. If it is, add text/plain as its MIME type (since it doesn't exist in the Lord of the Files MIME list yet). Once it gets added this conditional statement can be removed.

Reply:
Lord of the Files latest release (1.4.0) includes nlogo, so I removed the conditional here.